chain/epoch-manager/src/tests/mod.rs

```diff
@@ -1889,11 +1878,9 @@ fn test_protocol_version_switch() {
         EpochManager::new(store, config, reward_calculator, validators).unwrap();
     let h = hash_range(8);
     record_block(&mut epoch_manager, CryptoHash::default(), h[0], 0, vec![]);
-    let mut block_info1 = block_info(h[1], 1, 1, h[0], h[0], h[0], vec![], DEFAULT_TOTAL_SUPPLY);
-    set_block_info_protocol_version(&mut block_info1, 0);
-    epoch_manager.record_block_info(block_info1, [0; 32]).unwrap();
-    for i in 2..6 {
-        record_block(&mut epoch_manager, h[i - 1], h[i], i as u64, vec![]);
+    for i in 1..6 {
+        let version = if i == 1 { 0 } else { PROTOCOL_VERSION };
+        record_block_with_version(&mut epoch_manager, h[i - 1], h[i], i as u64, vec![], version);
```

Comment:
this seemed like a small enough refactor / improvement to include here and in the following 2 tests.